unturned/OpenMod.Unturned/Vehicles/Events/VehicleEventsListener.cs

```diff
@@ -56,6 +59,9 @@ public override void Unsubscribe()
             OnVehicleExited -= Events_OnVehicleExited;
             OnVehicleSwapped -= Events_OnVehicleSwapped;
             OnVehicleStealBattery -= Events_OnVehicleStealingBattery;
+            OnVehicleReplacingBattery -= Events_OnVehicleReplacingBattery;
+            OnVehicleLockUpdating += Events_OnVehicleLockUpdating;
```

Comment:
Unsubscribe instead of a second subscription

Reply:
Oops, ill try to me more careful with this.